lumicks/pylake/force_calibration/tests/test_power_spectrum.py

```diff
@@ -19,21 +19,21 @@ def test_blocking_separate(data, num_blocks, avg, std):
 
 
 @pytest.mark.parametrize(
-    "frequency, num_data, sampling_rate",
+    "frequency, num_data, sample_rate",
     [
         [200, 50, 2000],
         [400, 50, 2000],
         [400, 50, 4000],
         [400, 50, 4000],
     ],
 )
-def test_power_spectrum_attrs(frequency, num_data, sampling_rate):
+def test_power_spectrum_attrs(frequency, num_data, sample_rate):
     """Testing the attributes of power spectra"""
-    data = np.sin(2.0 * np.pi * frequency / sampling_rate * np.arange(num_data))
-    power_spectrum = PowerSpectrum(data, sampling_rate)
+    data = np.sin(2.0 * np.pi * frequency / sample_rate * np.arange(num_data))
+    power_spectrum = PowerSpectrum(data, sample_rate)
 
-    df = sampling_rate / num_data
-    frequency_axis = np.arange(0, .5 * sampling_rate + 1, df)
+    df = sample_rate / num_data
+    frequency_axis = np.arange(0, .5 * sample_rate + 1, df)
```

Comment:
super nit-picky but I would use `0.5` instead of `.5` as the decimal is easy to miss as far as readability

Reply:
Fixed! :)